### ISSUE TITLE
with default setting, LoRA has no improvement is normal? 默认设置下，LoRA 完全没效果正常吗？

### ISSUE THREAD

**Issue by fisher75** [sgl-project/sglang#396]
### Question

Hi, I used Lora for a 1.6-7b and 1.5-7b model, the training samples are about 100,000 images, and after I Lora I did an inference with the new Lora model. But it turned out to be no improvement at all (almost, like only a 1% improvement for 1.6 and 2% for 1.5). I think this is absolutely not normal cause in the same time I also used ViT and CLIIP as baselines to compare and are all trained too, they both have shown very significant improvement and until like 90%+ accuracy. So I want know is this normal and what might the problem be. And one more question, does the LoRA only affect on LLM or on whole VLM including the CLIP part?

Here I show my train command and inference command:
```
deepspeed llava/train/train_mem.py \
    --lora_enable True --lora_r 128 --lora_alpha 256 --mm_projector_lr 2e-5 \
    --deepspeed ./scripts/zero3.json \
    --model_name_or_path model/llava-v1.6-vicuna-7b \
    --version v1 \
    --data_path train/lora.json \
    --image_folder ./playground/data \
    --vision_tower model/clip-vit-large-patch14-336 \
    --mm_projector_type mlp2x_gelu \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --image_aspect_ratio pad \
    --group_by_modality_length True \
    --bf16 True \
    --output_dir lava-v1.6-vicuna-7b-ora \
    --num_train_epochs 1 \
    --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 50000 \
    --save_total_limit 1 \
    --learning_rate 2e-4 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 4 \
    --lazy_preprocess True \
    --report_to wandb
```

and

```
python -m llava.eval.model_vqa \
    --model-path lora/llava-v1.6-vicuna-7b-lora \
    --model-base model/llava-v1.6-vicuna-7b \
    --question-file questions.jsonl \
    --image-folder data \
    --answers-file lora_answers.jsonl \
    --temperature 0 \
    --conv-mode vicuna_v1
```

Thanks for your attention.
